Add a dockerfile for the DTL

smartcontracts opened the pull request:
Adds a new Dockerfile for the data transport layer. Q: What's the best way to confirm that this works correctly? It definitely builds but need to make sure it runs.

smartcontracts committed:
Add a dockerfile for the DTL

tynes commented on data-transport-layer/Dockerfile:
```diff
@@ -0,0 +1,27 @@
+
+FROM node:10.23-buster
+RUN apt-get update \
+    && apt-get install -y bash curl jq python3 libusb-1.0 \
```
Nit: I don't think we technically need python3 or libusb-1.0 here as well as libusb-1.0 in the above. I'm on with leaving libusb but getting rid of python3 means we can get rid of the symlink below. It was used for python simple http server in the deployer to start a server after the deployment was done, so that a `wait-for...` script could ping for the deployment to be finished

smartcontracts replied:
Oh nice, I missed that one.

smartcontracts committed:
Removed unnecessary dependencies

We had various dependencies in the dockerfile that I copied that we don't actually need for this flow (`python`, `libusb`).
